src/fsharp/service/service.fs

```diff
@@ -59,7 +59,7 @@ module EnvMisc =
     let maxTypeCheckErrorsOutOfProjectContext = GetEnvInteger "FCS_MaxErrorsOutOfProjectContext" 3
     let braceMatchCacheSize = GetEnvInteger "FCS_BraceMatchCacheSize" 5
     let parseFileCacheSize = GetEnvInteger "FCS_ParseFileCacheSize" 2
-    let checkFileInProjectCacheSize = GetEnvInteger "FCS_CheckFileInProjectCacheSize" 5
+    let checkFileInProjectCacheSize = GetEnvInteger "FCS_CheckFileInProjectCacheSize" 10
```

Comment:
@dsyme note that I've increased the stale caches sizes here because if a file is included into more than 5 projects, the cache effectively does not work at all (for example the TP SDK solution where ProvidedTypes.fs/fsi are included to several projects). I'm not sure how much memory it will occupy though.

Reply:
This seems ok, I don't think this cache is a memory hog

Reply:
Well, we'll find out with a profiler if that's the case 😄 

That said, this problems - files shared across multiple projects - may be unique to the TPSDK. I'm perfectly fine with the change so long as it spends time with people banging on it (e.g., in nightlies) and/or profiling reveals increasing it doesn't really affect memory usage.